src/nb_clean/__init__.py

```diff
@@ -86,8 +87,10 @@ def add_git_filter(
     ----------
     remove_empty_cells : bool, default False
         If True, remove empty cells.
-    preserve_cell_metadata : bool, default False
-        If True, preserve cell metadata.
+    preserve_cell_metadata : list of str or None, default None
+        If None, clean all cell metadata.
+        If [], preserve cell metadata.
```

Comment:
What does this imply?

Reply:
Now I understand that `[]` used to behave when `--preserve-cell-metadata` is given without any arguments.

Reply:
Maybe edit to clearly state "all cell metadata"?

Reply:
Agreed, that would be better. I'll also try to clarify the reason `[]` has this behavior.